misc/alert_rules.json

```diff
@@ -81,7 +81,7 @@
     "name": "IPSec tunnels down"
   },
   {
-    "rule": "%pollers.time_taken >= \"250\"",
+    "rule": "devices.last_polled_timetaken >= 290",
```

Comment:
Actually shouldn't this be %devices.last_polled_timetaken?

Reply:
We strip out the % before processing anyway. Did you try it?

Reply:
No but all of the others have the % in. I'm sure I stripped them out when I did the large alerting PR but seem to recall you added them back in maybe?

Reply:
Might have gotten reverted at some point.

Reply:
Also, this changes the alert from the pollers total time to a single device which is fine as the original one obviously won't work but it also needs the name of the rule updating.